docs/using_eessi/eessi_container.md

```diff
@@ -0,0 +1,165 @@
+# Using the EESSI container
+
+This page provides all information and guides you through several examples
+illustrating to access the EESSI software stack via a small container provided by
+EESSI. The container only contains a minimal number tools such as the CernVM-FS
+client and some configuration data.
+
+### Prerequisites
+
+- Singularity 3.7.x or newer _or_ Apptainer x.y.z or newer (any version with
```

Comment:
Any Apptainer should work, so change `x.y.z` to `1.0.0`?

Reply:
Fixed by https://github.com/EESSI/docs/pull/100/commits/0eae16fe58096e7b995d7e7d2ddc3d9208fc2001